policy/daemon.go

```diff
@@ -85,6 +75,23 @@ func New(opts ...Option) (Daemon, error) {
 		}
 	}
 
+	// create fetchers
+	p.fetchers = make(map[string]Fetcher, len(p.athenzDomains))
+	for _, domain := range p.athenzDomains {
+		f := fetcher{
+			domain:        domain,
+			expireMargin:  p.expireMargin,
+			retryInterval: p.errRetryInterval,
+			retryMaxCount: 3,
```

Comment:
should we make it configurable?

Reply:
should be.
This PR? It is an interface change.

Reply:
can be another PR.

Reply:
continue in https://github.com/yahoojapan/athenz-authorizer/pull/50